deepchem/models/optimizers.py

```diff
@@ -550,6 +550,66 @@ def _create_jax_schedule(self):
                                        staircase=self.staircase)
 
 
+class LambdaLRWithWarmup(LearningRateSchedule):
+    """A learning rate scheduler supporting warmup."""
```

Comment:
Can you expand this with a usage example

Reply:
added usage example